src/firmware_new/boards/frankie_v1/io/drivetrain_unit.c

```diff
@@ -0,0 +1,43 @@
+#include "firmware_new/boards/frankie_v1/io/drivetrain_unit.h"
+
+#include <math.h>
+#include <stdlib.h>
+
+typedef struct DrivetrainUnit
```

Comment:
Nit:

In other parts of this PR the 'T' is capitalized in DriveTrain. Maybe change them to be consistent one way.

Reply:
Great point! I think it should be `Drivetrain` because it's a single word. Changed all cases to match this.